.hound.yml

```
@@ -0,0 +1,4 @@
+javascript:
+  enabled: false
+eslint:
+  enabled: true
```

[This Hound documentation](https://intercom.help/hound/en/articles/2138530-eslint) would imply we need to add a pointer to our own eslint configuration file. Did you find Hound worked without explicitly adding the path?
```suggestion
eslint:
  enabled: true
  config_file: .eslintrc.json
```

---

I can't say I am able to confirm either way.